### PR TITLE
New navigation content refresh 

### PR DESCRIPTION
**Pull request by ka0ula**
Changed all the content from the Getting Started section to align with Product new features

### CODE REVIEW

File: docs/getting_started/create_community.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+## Types of users 
+
+### Single-tenant users
+For single-tenant users, a workspace is automatically created upon subscribing to Sekoia. Users can only have one Workspace, which does not contain any communities.
+
+### Multi-tenant users
+For multi-tenant users, such as Managed Security Service Providers (MSSPs), a primary workspace is created. These users can add multiple communities to manage their various clients or departments.
+
+!!! info
+    To access multi-tenant mode, contact your Sekoia account executive. Once subscribed, you will be switched to muulti-tenant mode in the following days.
```
Comment (rombernier) [inline docs/getting_started/create_community.md:12]:
```suggestion
    To access multi-tenant mode, contact your Sekoia account executive. Once subscribed, you will be switched to multi-tenant mode in the following days. 
```

Comment (rombernier) [reply docs/getting_started/create_community.md:12]: A sub-community cannot be switched to a multi-tenant mode while remaining a sub-community. Perhaps we should specify this.

**Comment by Sengthay** [comment]
The menu still displays "Create and delete communities" instead of "edit".

image

**Comment by ka0ula** [comment]
> The menu still displays "Create and delete communities" instead of "edit".
> 
> image

Fixed 👍

**Review by Sengthay — APPROVED** [review]

**Pull request merged**